crates/ruma-api/src/lib.rs

```diff
@@ -421,3 +425,129 @@ pub struct Metadata {
     /// What authentication scheme the server uses for this endpoint.
     pub authentication: AuthScheme,
 }
+
+/// The matrix versions ruma currently understands to exist.
+///
+/// Matrix, since fall 2021, has a quarterly release schedule, using a global `vX.Y` versioning
+/// scheme.
+///
+/// Every new minor version denotes stable support for endpoints in a *relatively*
+/// backwards-compatible manner.
+///
+/// Matrix has a deprecation policy, read more about it here; https://spec.matrix.org/v1.2/#deprecation-policy
```

Comment:
Sentences should end in a priod, and as per rustdoc warning, this link won't be rendered as a link.

```suggestion
/// Matrix has a deprecation policy, read more about it here: <https://spec.matrix.org/v1.2/#deprecation-policy>.
```